src/libraries/System.Reflection.Emit/tests/AssemblyBuilderTests.cs

```diff
@@ -424,7 +424,16 @@ void Invoke_Private_SameAssembly_ThrowsMethodAccessException()
 	    Assert.Throws<MethodAccessException>(() => d ());
 	}
 
+    [Fact]
+    public void DefineDynamicAssembly_InternalAssemblyLocationIsEmpty()
+    {
+        AssemblyBuilder assembly = Helpers.DynamicAssembly(nameof(DefineDynamicAssembly_InternalAssemblyLocationIsEmpty));
+        Assembly internalAssemblyBuilder  = AssemblyLoadContext.Default.Assemblies
+                .FirstOrDefault(_ => _.FullName == assembly.FullName);
```

Comment:
```suggestion
                .FirstOrDefault(a => a.FullName == assembly.FullName);
```

Don't use `_` for a variable name if you are actually going to use the variable. `_` is used when you want to ignore the variable/return value.